package.json

```diff
@@ -132,24 +132,27 @@
     "rxjs-marbles": "2.3.1",
     "source-map-loader": "0.1.5",
     "string-replace-webpack-plugin": "0.0.3",
-    "style-loader": "0.13.1",
+    "style-loader": "0.20.1",
     "stylelint": "7.2.0",
     "stylelint-config-dcos": "0.0.3",
     "stylelint-config-standard": "13.0.0",
     "svg-sprite": "1.3.1",
     "transform-loader": "0.2.3",
+    "uglifyjs-webpack-plugin": "1.2.0",
     "vinyl": "1.1.1",
-    "webpack": "1.14.0",
-    "webpack-dev-server": "1.14.1",
-    "webpack-notifier": "1.3.0"
+    "webpack": "3.10.0",
```

Comment:
Why webpack v3 and not v4?

Reply:
Because 4 is so new that our dependencies haven't updated yet. Also most of the work was done already before 4 came out, so I prioritized ship it here.
I plan to revisit this in ~ 1 month. The API changes in webpack were only severe for the packages and not for the configuration, so upgrading should work smoothly once the ecosystem adopted